dist/init/linux-upstart/caddy.conf

```diff
@@ -1,18 +1,25 @@
-description "Caddy startup script"
+description "Caddy HTTP/2 web server"
 author "Mathias Beke"
 
 start on runlevel [2345]
 stop on runlevel [016]
 
+console logged
 
 setuid www-data
 setgid www-data
 
 respawn
 respawn limit 10 5
 
-limit nofile 4096 4096
+# Letsencrypt-issued certificates will be written to this directory.
+env HOME=/etc/ssl/caddy
+
+limit nofile 1048576 1048576
 
 script
-	exec /usr/bin/caddy -agree=true -conf=/etc/caddy/Caddyfile
-end script
\ No newline at end of file
+	mkdir -p /etc/caddy
+	mkdir -p /var/tmp/caddy
+	cd /etc/caddy
+	exec /usr/bin/caddy -agree=true -email -log=stdout -conf=/etc/caddy/Caddyfile -root=/var/tmp/caddy
```

Comment:
I've been thinking about this scenario: »I didn't agree to anything. Oh, this is not me, it's been shipped this way by whoever maintains the PPA.«

IMHO the process needs the operator's consideration, and he needs to set at least one thing himself to show he understood that he is consenting into a binding agreement.

For Linux distros, *init* files should be close to immutable (some distros even ship them read-only! (like CoreOS)), and everything individual is expected to become part of configuration files.